selfdrive/ui/qt/maps/map.cc

```diff
@@ -576,9 +576,9 @@ void MapInstructions::updateDistance(float d) {
   distance->setText(distance_str);
 }
 
-void MapInstructions::showError(QString error) {
+void MapInstructions::showError(QString error_text) {
```

Comment:
This was an actual bug!

Reply:
Ran into it in https://github.com/commaai/openpilot/pull/22767

Reply:
haha good to hear!